client/app/bundles/course/assessment/submission/pages/SubmissionsIndex/__test__/submissionsTable.test.jsx

```diff
@@ -0,0 +1,78 @@
+import React from 'react';
+import { mount } from 'enzyme';
+import ProviderWrapper from 'lib/components/ProviderWrapper';
+import SubmissionsTable from '../SubmissionsTable';
+
+const setupAssessmentProps = propsOverrides => Object.assign(
+  {
+    canViewLogs: true,
+    downloadable: true,
+    gamified: true,
+    maximumGrade: 10,
+    passwordProtected: true,
+    title: 'Password Protected Exam',
+  },
+  propsOverrides
+);
+
+const setupTest = (propsOverrides) => {
+  const props = Object.assign({
+    assessment: setupAssessmentProps(),
+    courseId: '1',
+    assessmentId: '1',
+    isDownloading: false,
+    isStatisticsDownloading: false,
+    submissions: [
+      {
+        id: 1,
+        workflowState: 'submitted',
+        grade: 10,
+        pointsAwarded: 100,
+        dateSubmitted: '2018-03-08T18:46:59.292+08:00',
+        dateGraded: '2018-04-08T18:46:59.292+08:00',
+        courseStudent: {
+          id: 1,
+          myStudent: false,
+          name: 'John',
+          path: '/foo',
+          phantom: false,
+        },
+      },
+    ],
+  }, propsOverrides);
+
+  const submissionsTable = mount(
+    <ProviderWrapper>
+      <SubmissionsTable {...props} />
+    </ProviderWrapper>
+  );
+
+  return {
+    props,
+    submissionsTable,
+    rowCount: submissionsTable.find('tr.submission-row'),
+    logCount: submissionsTable.find('div.submission-access-logs'),
+  };
+};
+
+describe('<SubmissionsTable />', () => {
+  describe('when canViewLogs is set to true ', async () => {
+    it('renders the submissions table with access log links', async () => {
+      const assessmentProps = setupAssessmentProps({ canViewLogs: true });
```

Comment:
I think it's cleaner to use `...` instead of `Object.assign` here and for `setupTest`, as in:

```js
const defaultAssessmentProps = {
  canViewLogs: true,
  downloadable: true,
  gamified: true,
  maximumGrade: 10,
  passwordProtected: true,
  title: 'Password Protected Exam',
}

//

const assessmentProps = { ...defaultAssessmentProps,  canViewLogs: true }

```